src/approaches/random_options_approach.py

```diff
@@ -0,0 +1,43 @@
+"""An approach that just executes random options.
+"""
+
+from collections import defaultdict
+from typing import Callable, List
+from predicators.src.approaches import BaseApproach
+from predicators.src.structs import State, Task, Action
+from predicators.src.settings import CFG
+from predicators.src import utils
+
+
+class RandomOptionsApproach(BaseApproach):
+    """Samples random options (and random parameters for those options).
+    """
+    @property
+    def is_learning_based(self) -> bool:
+        return False
+
+    def _solve(self, task: Task, timeout: int) -> Callable[[State], Action]:
+        options = sorted(self._initial_options, key=lambda o: o.name)
+        plan: List[Action] = []
+        def _policy(state: State) -> Action:
+            nonlocal plan
+            if plan:
+                return plan.pop(0)
+            for _ in range(CFG.random_options_max_tries):
+                param_opt = options[self._rng.choice(len(options))]
+                types_to_objs = defaultdict(list)
```

Comment:
worth moving this logic into a utility function? seems potentially generally useful (sample a random object combination from a state, given a list of types)